File: programmers/js/level00/120829.js
// 각도기
function solution(angle) {
  if (angle == 180) {
    return 4;
  } else if (angle > 90) {
    return 3;
  } else if (angle == 90) {
    return 2;
  } else if (90 > angle) {
    return 1;
  }
}

// 다른 풀이
// function solution(angle) {
//   return [0, 90, 91, 180].filter(x => angle >= x).length;
// }

File: programmers/python/level00/120829.py
# 각도기
def solution(angle):
    if angle == 180:
        return 4
    elif 180 > angle > 90:
        return 3
    elif angle == 90:
        return 2
    elif 90 > angle > 0:
        return 1

# 다른 풀이
# def solution(angle):
#     return (angle // 90) * 2 + (angle % 90 > 0) * 1